hugo/layouts/partials/species_nav.html

```diff
@@ -13,14 +13,16 @@
 
     </li>
     {{ range where .CurrentSection.Pages "Section" $currentSection }}
-    <li class="nav-item">
 
-        {{ if eq $current_page .Page }}
-            <a class="nav-link active" href="{{ .RelPermalink }}"> {{ .Name }} </a>
-        {{ else }}
-            <a class="nav-link" href="{{ .RelPermalink }}"> {{ .Name }} </a>
+        {{ if ne .Name "Browser" }}
```

Comment:
Is this necessary, since `species_browser.html` was deleted ?

Reply:
Good point. I've removed it now in this commit: [79c3ce3](https://github.com/ScilifelabDataCentre/swedgene/pull/11/commits/79c3ce38d31bda8ab8fb6c7983be4644bb4441ab)